spec/03-plugins/24-response-rate-limiting/04-access_spec.lua

```diff
@@ -9,20 +9,64 @@ local REDIS_SSL_SNI   = helpers.redis_ssl_sni
 local REDIS_PASSWORD  = ""
 local REDIS_DATABASE  = 1
 
-local SLEEP_TIME = 0.01
-local ITERATIONS = 10
+local ITERATIONS = 6
+local escape_uri = ngx.escape_uri
+local encode_args = ngx.encode_args
 
 local fmt = string.format
 
 
 local proxy_client = helpers.proxy_client
 
 
-local function wait()
+-- wait for server timestamp reaching the ceiling of client timestamp secs
+-- e.g. if the client time is 1.531 secs, we want to start the test period
+-- at 2.000 of server time, so that we could have as close as 1 sec to 
+-- avoid flaky caused by short period(e.g. server start at 1.998 and it soon
+-- exceed the time period)
+local function wait_server_sync(headers, api_key)
```

Comment:
The response rate limiting period is maintain by the server side(kong). In the previous `wait`, we just wait on the client side and the wait logic could not guarantee server side start a new second, sometimes when the wait end, server side is still in the tail of old second.
Here we get the timestamp from server, wait until server starts a new second so that is as as close as 1 secs to finish the test iterations, reduce the possibility of flaky.